t/rosa-svn-pre-commit/04-aliases.t

```diff
@@ -0,0 +1,298 @@
+sed -i "s/owner=daisy/owner=daisynew/" aa000/rose-suite.info
+sed -i "s/access-list=frank dave/access-list=franknew davenew/" aa000/rose-suite.info
+REV1=$(svn info --show-item revision aa000/)
+run_pass "$TEST_KEY" svn commit -q -m 't' --username=franknew aa000
+file_cmp "$TEST_KEY.out" "$TEST_KEY.out" </dev/null
+file_cmp "$TEST_KEY.err" "$TEST_KEY.err" </dev/null
+svn update -q aa000
+REV2=$(svn info --show-item revision aa000/)
+run_pass "$TEST_KEY.rev" bash -c "(( $REV2 > $REV1 ))"
+svnlook changed repos/foo >"$TEST_KEY.changed"
+file_cmp "$TEST_KEY.changed" "$TEST_KEY.changed" <<'__CHANGED__'
+U   a/a/0/0/0/trunk/rose-suite.info
+__CHANGED__
+#-------------------------------------------------------------------------------
+TEST_KEY=$TEST_KEY_BASE-modify-with-alias-access-list-bad-1
+# Restore the state without aliases.
+svn update -q aa000
+sed -i "s/new//g" aa000/rose-suite.info
+REV1=$(svn info --show-item revision aa000/)
+run_pass "$TEST_KEY-setup" svn commit -q -m 't' --username=daisynew aa000
+file_cmp "$TEST_KEY-setup.out" "$TEST_KEY-setup.out" </dev/null
+file_cmp "$TEST_KEY-setup.err" "$TEST_KEY-setup.err" </dev/null
+svn update -q aa000
+REV2=$(svn info --show-item revision aa000/)
+run_pass "$TEST_KEY-setup.rev" bash -c "(( $REV2 > $REV1 ))"
+svnlook changed repos/foo >"$TEST_KEY-setup.changed"
+file_cmp "$TEST_KEY-setup.changed" "$TEST_KEY-setup.changed" <<'__CHANGED__'
+U   a/a/0/0/0/trunk/rose-suite.info
+__CHANGED__
+# Access-list user trying to change the owner, should be blocked.
+sed -i "s/owner=daisy/owner=franknew/" aa000/rose-suite.info
+sed -i "s/access-list=frank dave/access-list=franknew davenew/" aa000/rose-suite.info
+REV1=$(svn info --show-item revision aa000/)
+run_fail "$TEST_KEY" svn commit -q -m 't' --username=franknew aa000
+file_cmp "$TEST_KEY.out" "$TEST_KEY.out" </dev/null
+file_cmp "$TEST_KEY.err" "$TEST_KEY.err" <<'__ERR__'
+svn: E165001: Commit failed (details follow):
+svn: E165001: Commit blocked by pre-commit hook (exit code 1) with output:
+[FAIL] PERMISSION DENIED: U   a/a/0/0/0/trunk/rose-suite.info: owner=franknew
+
+__ERR__
+#-------------------------------------------------------------------------------
+TEST_KEY=$TEST_KEY_BASE-modify-with-alias-access-list-bad-2
+svn revert -q -R aa000
+# Access-list user trying to change the access-list, should be blocked.
+sed -i "s/owner=daisy/owner=daisynew/" aa000/rose-suite.info
+sed -i "s/access-list=frank dave/access-list=franknew davenew heywood/" aa000/rose-suite.info
+REV1=$(svn info --show-item revision aa000/)
+run_fail "$TEST_KEY" svn commit -q -m 't' --username=franknew aa000
+file_cmp "$TEST_KEY.out" "$TEST_KEY.out" </dev/null
+file_cmp "$TEST_KEY.err" "$TEST_KEY.err" <<'__ERR__'
+svn: E165001: Commit failed (details follow):
+svn: E165001: Commit blocked by pre-commit hook (exit code 1) with output:
+[FAIL] PERMISSION DENIED: U   a/a/0/0/0/trunk/rose-suite.info: access-list=davenew franknew heywood
```

Comment:
FYI this is the new error message following our access-list user error change, think is good.